### DISCUSSION TITLE
config.yaml

### DISCUSSION THREAD

**Discussion by rmica** [Chia-Network/chia-blockchain#7362] [Support]
Both farmer and pool sections of config.yaml say:
...
\# Replace this with a real puzzle hash
\# xch_target_address: txch102gkhhzs60grx7cfnpng5n6rjecr89r86l5s8xux2za8k820cxsq64ssdg
...

Do I have to enter an address or puzzle hash format?

Thank you for help.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
